src/pages/task/taskTabs/FileTable/index.tsx

```diff
@@ -0,0 +1,74 @@
+import { useState } from "react";
+import { useQuery } from "@apollo/client";
+import styled from "@emotion/styled";
+import { SearchInput } from "@leafygreen-ui/search-input";
+import { Skeleton, TableSkeleton } from "@leafygreen-ui/skeleton-loader";
+import { Body } from "@leafygreen-ui/typography";
+import { size } from "constants/tokens";
+import { useToastContext } from "context/toast";
+import { TaskFilesQuery, TaskFilesQueryVariables } from "gql/generated/types";
+import { GET_TASK_FILES } from "gql/queries";
+import GroupedFilesTable from "./GroupedFilesTable";
+import { filterGroupedFiles } from "./utils";
+
+interface FilesTableProps {
+  taskId: string;
+  execution: number;
+}
+const FilesTable: React.FC<FilesTableProps> = ({ execution, taskId }) => {
```

Comment:
[nit] The file is called `FileTable` but the component is called `FilesTable`, would prefer to be consistent!